filer/static/filer/js/addons/dropzone.init.js

```diff
@@ -9,60 +9,75 @@
         var previewImageSelector = '.js-img-preview';
         var dropzoneSelector = '.js-dropzone';
         var messageSelector = '.js-dropzone-message';
-        var lookupSelector = '.js-related-lookup';
+        var lookupButtonSelector = '.js-related-lookup';
         var progressSelector = '.js-dropzone-progress';
         var previewImageWrapperSelector = '.js-img-wrapper';
         var filerClearerSelector = '.filerClearer';
-        var dropzone = $(dropzoneSelector);
-        var dropzoneUrl = dropzone.data('url');
+        var dropzones = $(dropzoneSelector);
         var fileSelector = $('.js-file-selector');
-        var fileIdInputSelector = $('#id_file');
+        var fileIdInputSelector = '.vForeignKeyRawIdAdminField';
         var hiddenClass = 'hidden';
 
-        if (dropzone.length && Dropzone) {
+        if (dropzones.length && Dropzone && !window.initEnabled) {
+            window.initEnabled = true;
```

Comment:
if there is any way to not rely on global variable - would be great, otherwise at least make it as specific as possible (e.g. filerBlaBlaInitialized)